vxdetector/Output_counter.py

```diff
@@ -18,58 +22,120 @@ def directory_navi(file_name, path, dir_name, dir_path):
     return file_name, dir_name, dir_path
 
 
-def create_output(path, file_name, unaligned_count, most_probable_V,
-                  probability, dir_name, dir_path):
+def rawincount(filename):
+    f = open(filename, 'rb')
+    bufgen = takewhile(lambda x: x, (f.raw.read(1024*1024)
+                                     for _ in repeat(None)))
+
+    return sum(buf.count(b'\n') for buf in bufgen)
+
+
+def region_count(dictionary, unaligned_count, temp_path, all_reads, mode):
+    if temp_path is not None and mode == 'paired':
+        unpaired = round((rawincount(f'{temp_path}bed.log')/all_reads), 2)
+    elif mode == 'unpaired':
+        unpaired = 'unpaired Reads'
+    no_V = rawincount(f'{temp_path}noOver.bed')
+    aligned_count = 100 - unaligned_count
+    count = sum([dictionary['V1'], dictionary['V2'], dictionary['V3'],
+                dictionary['V4'], dictionary['V5'], dictionary['V6'],
+                dictionary['V7'], dictionary['V8'], dictionary['V9'], no_V])
+    dictionary['V1'] = round((dictionary['V1'] / count) * aligned_count, 2)
```

Comment:
we should differentiate between internal computation (= no need to chomp decimal places) and user report (= no need to show more than 2 digits)